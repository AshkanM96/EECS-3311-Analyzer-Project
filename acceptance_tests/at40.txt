for_all
type_check
evaluate
open_set_enumeration
type_check
evaluate
close_set_enumeration
type_check
evaluate
reset
for_all
open_set_enumeration
boolean_constant(true)
boolean_constant(true)
boolean_constant(true)
type_check
evaluate
negation
boolean_constant(true)
close_set_enumeration
type_check
evaluate
