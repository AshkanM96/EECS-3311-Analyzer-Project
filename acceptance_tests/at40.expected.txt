  Expression currently specified: ?
  Report: Expression is initialized.
->for_all
  Expression currently specified: (&& ?)
  Report: OK.
->type_check
  Expression currently specified: (&& ?)
  Report: Error (Expression is not yet fully specified).
->evaluate
  Expression currently specified: (&& ?)
  Report: Error (Expression is not yet fully specified).
->open_set_enumeration
  Expression currently specified: (&& {?})
  Report: OK.
->type_check
  Expression currently specified: (&& {?})
  Report: Error (Expression is not yet fully specified).
->evaluate
  Expression currently specified: (&& {?})
  Report: Error (Expression is not yet fully specified).
->close_set_enumeration
  Expression currently specified: (&& {?})
  Report: Error: (Set enumeration must be non-empty).
->type_check
  Expression currently specified: (&& {?})
  Report: Error (Expression is not yet fully specified).
->evaluate
  Expression currently specified: (&& {?})
  Report: Error (Expression is not yet fully specified).
->reset
  Expression currently specified: ?
  Report: OK.
->for_all
  Expression currently specified: (&& ?)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (&& {?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, ?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, True, ?})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, True, True, ?})
  Report: OK.
->type_check
  Expression currently specified: (&& {True, True, True, ?})
  Report: Error (Expression is not yet fully specified).
->evaluate
  Expression currently specified: (&& {True, True, True, ?})
  Report: Error (Expression is not yet fully specified).
->negation
  Expression currently specified: (&& {True, True, True, (! ?)})
  Report: OK.
->boolean_constant(True)
  Expression currently specified: (&& {True, True, True, (! True), ?})
  Report: OK.
->close_set_enumeration
  Expression currently specified: (&& {True, True, True, (! True)})
  Report: OK.
->type_check
  Expression currently specified: (&& {True, True, True, (! True)})
  Report: (&& {True, True, True, (! True)}) is type-correct.
->evaluate
  Expression currently specified: (&& {True, True, True, (! True)})
  Report: False
